oozie/oozie.sh

```diff
@@ -679,6 +697,10 @@ EOF
 
 
 function main() {
+  #Remove debian backports
+  if [[ ${OS_NAME} == debian ]]; then 
```

Comment:
Adding a check for dataproc version is also required, as the issue only affected debian 10 which is being used by dataproc 2.0

Reply:
Done, added condition.